### DISCUSSION TITLE
Question about the plugin

### DISCUSSION THREAD

**Discussion by seloner** [jscutlery/semver#224] [Q&A] [answered]
If I have two projects a and b.
I commit one feature for project a and one feature for project b.
Then I version project a .
Will the changelog for project a have the changes of b?
Should I work in only one project and then version in order to not merge changelogs?

**Comment by edbzn** [comment, marked as answer]
Hi, it depends on the mode:

- with sync mode, there are project changelogs and one root changelog containing all workspace changes, note that you can configure both.
- with independent mode, there are only project changelogs.

> Will the changelog for project a have the changes of b?

No, except the root changelog when using sync mode.

> Should I work in only one project and then version in order to not merge changelogs?

If you want to manage each project independently of each other, maybe independent mode is what you need.
